### PR TITLE
Update FirebaseFunctions JavaDoc

### PR DESCRIPTION
**Pull request by samtstern**
See http://cl/347916042 for context

### CODE REVIEW

File: firebase-functions/src/main/java/com/google/firebase/functions/FirebaseFunctions.java [hunk 1]
```diff
@@ -157,7 +157,7 @@ public void onProviderInstallFailed(int i, android.content.Intent intent) {
    *
    * @param app The app for the Firebase project.
    * @param regionOrCustomDomain The region or custom domain for the HTTPS trigger, such as
-   *     "us-central1" or "https://mydomain.com".
+   *     <code>"us-central1"</code> or <code>"https://mydomain.com"</code>.
```
Comment (rlazo) [inline firebase-functions/src/main/java/com/google/firebase/functions/FirebaseFunctions.java:160]:
```suggestion
   *     {@code "us-central1"} or {@code "https://mydomain.com"}.
```

File: firebase-functions/src/main/java/com/google/firebase/functions/FirebaseFunctions.java [hunk 2]
```diff
@@ -184,8 +184,8 @@ public static FirebaseFunctions getInstance(@NonNull FirebaseApp app) {
   /**
    * Creates a Cloud Functions client with the default app and given region or custom domain.
    *
-   * @param regionOrCustomDomain The regionOrCustomDomain for the HTTPS trigger, such as
-   *     "us-central1" or "https://mydomain.com".
+   * @param regionOrCustomDomain The region or custom domain for the HTTPS trigger, such as
+   *     <code>"us-central1"</code> or <code>"https://mydomain.com"</code>.
```
Comment (rlazo) [inline firebase-functions/src/main/java/com/google/firebase/functions/FirebaseFunctions.java:188]:
```suggestion
   *     {@code "us-central1"} or {@code "https://mydomain.com"}.
```

**Review by rlazo — APPROVED** [review]

**Comment by rlazo** [comment]
BTW the test is failing because of `:firebase-functions:verifyGoogleJavaFormat`. A `./gradlew firebase-functions:gJF` should do the trick.

**Review by Padil77 — APPROVED** [review]

**Review by Padil77 — APPROVED** [review]

**Comment by rlazo** [comment]
It's safe to ignore the smoke-tests. The failure is because

```
com.google.firebase.FirebaseTooManyRequestsException: This project's quota for this operation has been exceeded. [
```

**Pull request merged**